pegasus/sites.v3/code.org/public/donate/embedded_experiment.haml

```diff
@@ -0,0 +1,22 @@
+---
+title: Donate Experiment
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/donate-styles.css", rel: "stylesheet"}
+
+-# Optional: Classy tracking pixel
+= view :classy_tracking_pixel
```

Comment:
I think all of this can be removed. The tracking pixel isn't relevant for this experiment, it's just a marketing thing.